docs/src/pages/components/breadcrumbs/RouterBreadcrumbs.js

```diff
@@ -101,18 +101,18 @@ class RouterBreadcrumbs extends React.Component {
               );
             }}
           </Route>
-          <div className={classes.lists}>
-            <List component="nav">
+          <nav className={classes.lists}>
```

Comment:
`nav` should have `aria-label` (or `aria-labelledby` if there's associated visible text)

```suggestion
          <nav className={classes.lists} aria-label="Mailbox folders">
```

Reply:
Yep: https://www.w3.org/TR/wai-aria-practices/#aria_lh_step3

Will have a look why lighthouse doesn't report those.

Reply:
And look through all the landmark usages in the demos.